lib/msf/core/optional_session.rb

```diff
@@ -37,8 +38,11 @@ def initialize(info = {})
           Msf::OptInt.new('SESSION', [ false, 'The session to run this module on' ]),
           Msf::OptString.new('DATABASE', [ false, 'The database to authenticate against', 'postgres']),
           Msf::OptString.new('USERNAME', [ false, 'The username to authenticate as', 'postgres']),
+          Msf::Opt::RHOST(nil, false),
+          Msf::Opt::RPORT(5432, false)
```

Comment:
For context, these options were initially added here: https://github.com/rapid7/metasploit-framework/commit/23a9938283a503f9fd37e823af0211d92bc2e59b#diff-5479394bb6dd61d2ac4fc96ae36d29cf5b52e3b7687b7b6c714cbe53b273c4cb
But seemed to be removed here: https://github.com/rapid7/metasploit-framework/commit/2c60780dc06ad48b086523cc4261f7314317e0ea#diff-5479394bb6dd61d2ac4fc96ae36d29cf5b52e3b7687b7b6c714cbe53b273c4cb